File: cs/Iris/ParserUtils/LexParseException/EmptyExpression.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class EmptyExpression : BaseException
    {
        public EmptyExpression(int lNum, int cNum)
            : base(lNum, cNum)
        {

        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append("empty expression: ");
            str.Append(base.stringTrace());

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/EmptyExpression.java
package com.rokuan.iris.exception;

public class EmptyExpression extends BaseException
{
	public EmptyExpression(int lNum, int cNum)
	{
		super(lNum, cNum);
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append("empty Expression: ");
		str.append(super.stringTrace());

		return str.toString();
	}
}
